Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,338 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowGeodesicSectorAndEllipseView: View {
+    /// The data model that helps determine the view.
+    @StateObject private var model = Model()
+    
+    /// Manages the presentation state of the menu.
+    @State private var isPresented: Bool = false
+    
+    var body: some View {
+        MapViewReader { proxy in
+            MapView(
+                map: model.map,
+                graphicsOverlays: model.graphicOverlays
+            )
+            .onSingleTapGesture { _, tapPoint in
+                model.centerPoint = tapPoint
+            }
+            .task(id: model.centerPoint) {
+                guard let centerPoint = model.centerPoint else { return }
+                await proxy.setViewpoint(
+                    Viewpoint(center: centerPoint, scale: 1e7)
+                )
+            }
+            .toolbar {
+                // The menu which holds the options that change the ellipse and sector.
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Settings") {
+                        isPresented = true
+                    }
+                    .disabled(model.centerPoint == nil)
+                    .popover(isPresented: $isPresented) {
```

Comment:
Here is what this popover looks like on Mac Catalyst and visionOS:

![image](https://github.com/user-attachments/assets/6716d85c-dd6c-487e-9bea-794626c4c982)

Take a look at some of the other samples to see how they've addressed this. Also, make sure to test in both portrait and landscape orientations. This popover is presented as a sheet on an iOS device in landscape orientation. How is the user supposed to dismiss the sheet without rotating the device to portrait?

Reply:
I still need to test catalyst and visionOS

Reply:
<img width="1010" alt="Screenshot 2025-06-26 at 4 32 05 PM" src="https://github.com/user-attachments/assets/e24b0bd7-d8da-454b-862c-365badf83dda" />